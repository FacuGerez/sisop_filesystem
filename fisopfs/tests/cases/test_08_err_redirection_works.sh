#!/bin/bash
set -euo pipefail

MOUNT=tests/mount

cat "$MOUNT"/moms_spaghetti 2> "$MOUNT"/errorfile
ls "$MOUNT"
cat "$MOUNT"/errorfile