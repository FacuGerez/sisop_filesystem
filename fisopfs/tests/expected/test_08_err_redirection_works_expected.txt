errorfile
cat: moms_spaghetti: No such file or directory
